markdown/3.x/en/api/ff-communication.api.md

```diff
@@ -45,8 +45,8 @@ ___
 | **currency-min-digits**&nbsp;(String) | [Currency Guide](/documentation/3.x/currency-guide) |
 | **currency-max-digits**&nbsp;(String) | [Currency Guide](/documentation/3.x/currency-guide) |
 | **add-unit-to-filter-elements**&nbsp;(Boolean) `true` if present, else `false` | [Currency Guide](/documentation/3.x/currency-guide) |
-| **disable-single-hit-redirect**&nbsp;(String) **Options**: &nbsp;"true", &nbsp;"false" (default: "false") | Disables the automatic redirect to the product detail page if only one record is found |
-| **single-hit-redirect-base-path**&nbsp;(String) | Use this as an absolute base path if deeplinks are relative |
+| `(DEPRECATED)`**disable-single-hit-redirect**&nbsp;(String) **Options**: &nbsp;"true", &nbsp;"false" (default: "false") | Disables the automatic redirect to the product detail page if only one record is found |
+| `(DEPRECATED)` **single-hit-redirect-base-path**&nbsp;(String) | Use this as an absolute base path if deeplinks are relative |
```

Comment:
Not the format that is beiing used in other places, but I quite like it. Perhaps we should consider using it everywhere.

Reply:
should I adjust rest of deprecated things to this format?

Reply:
If you have the time, that would be great.

Reply:
I changed the:
`markdown/3.x/en/api/ff-breadcrumb-trail.api.md`
`markdown/3.x/en/api/ff-record-list.api.md`

I'm not sure about  `markdown/1.x/en/documentation/currency-guide.md:104`
If you ask me I'd keep it unchanged

Reply:
That's fine. No need to touch `1.x`